python/pip_install/extract_wheels/lib/BUILD

```diff
@@ -70,18 +70,22 @@ py_test(
     ],
 )
 
-py_test(
-    name = "whl_filegroup_test",
-    size = "small",
-    srcs = [
-        "whl_filegroup_test.py",
-    ],
-    data = ["//examples/wheel:minimal_with_py_package"],
-    tags = ["unit"],
-    deps = [
-        ":lib",
-    ],
-)
+# The `whl_filegroup_test` changes directories at runtime which impacts the
```

Comment:
could we just fix the test to restore the original conditions when it finishes, so it's safe to run them in the same interpreter?

Reply:
The test is lazily written IMO. `pwd` should not be a requirement for this functionality and patterns where you need to `chdir` are almost never maintainable. I think the correct fix is to refactor these all together but this was the quickest path forward for me. I could remove the sharding and the tests would work but there'd be a potential flake. I'd rather do that than a larger refactor.

Reply:
Ah, this ended up being (potentially) small 😄